colossalai/inference/core/rpc_engine.py

```diff
@@ -0,0 +1,274 @@
+import asyncio
+from itertools import count
+from time import sleep
+from typing import List, Tuple, Union
+
+import rpyc
```

Comment:
You might want to add any necessary dependency to `requirements/requirements.txt`, or at least `requirements/requirements-test.txt` to pass CI tests (Build-on-pr tests will install `requirements-test.txt`)

Reply:
Thx!